src/server/conn.rs

```diff
@@ -103,6 +103,7 @@ pub struct Http<E = Exec> {
     h1_keep_alive: bool,
     h1_title_case_headers: bool,
     h1_preserve_header_case: bool,
+    h1_header_read_timeout: Option<Duration>,
```

Comment:
[I can't leave the comment on an unchanged section of the file so I'll leave it here]:

`Duration` gets imported at the beginning of the file only if the `runtime` and `http2` features are on, so it doesn't work when just `http1` is on